modules/claims_api/app/swagger/claims_api/v2/dev/swagger.json

```diff
@@ -781,7 +797,7 @@
                           "openedDate": null,
                           "overdue": true,
                           "requestedDate": "2022-02-04",
-                          "status": "NEEDED",
+                          "status": "INITIAL_REVIEW_COMPLETE",
```

Comment:
assuming these are from the updates to the `receive_dt` PR yesterday that we just missed adding to docs.

Reply:
Odd, I see those changes in [API-24329-tracked-item-needed-status](https://github.com/department-of-veterans-affairs/vets-api/pull/12309/files#top)